### PR TITLE
Add additional metadata about deploys to ES, Don't split docs on multiple components

### PR DESCRIPTION
**Pull request by foklepoint**
Now that we have metadata being dispatched to ES, lets start sending more deploy metadata to elasticsearch

### CODE REVIEW

**Comment by foklepoint** [comment]
👓 @ckwang8128

File: rollingpin/elasticsearch.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        :return: Deferred
+        """
+        with swallow_exceptions("elasticsearch", self.logger):
+            agent = Agent(reactor)
+            body = JSONBodyProducer(json.dumps(doc))
+            return agent.request(
+                'POST',
+                self.endpoint,
+                Headers({'User-Agent': ['rollingpin']}),
+                body
```
Comment (ckwang8128) [inline rollingpin/elasticsearch.py]: nit: trailing comma

File: rollingpin/elasticsearch.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        now = int(time.time())
+        return {
+            'id': self.deploy_name,
+            'timestamp': now,
+            'components': self.components,
+            'deployer': getpass.getuser(),
+            'command': self.command_line,
+            'hosts': self.hosts,
+            'host_count': len(self.hosts),
+            'event_type': 'deploy.begin'
```
Comment (ckwang8128) [inline rollingpin/elasticsearch.py]: nit: trailing comma

File: rollingpin/elasticsearch.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import getpass
```
Comment (ckwang8128) [inline rollingpin/elasticsearch.py]: Module comment please

**Comment by ckwang8128** [comment]
:nail_care: cool! few small nits re: trailing commas (there are some I didn't specifically comment on) and a request for a elasticsearch.py module-level comment. can't wait to see this in action.

File: rollingpin/elasticsearch.py (hunk 3 above)
Comment (foklepoint) [inline rollingpin/elasticsearch.py]: Addressed

**Comment by foklepoint** [comment]
💇‍♂️

**Comment by ckwang8128** [comment]
:fish:

**Pull request merged**